app/controllers/movies_controller.rb

```diff
@@ -1,6 +1,24 @@
 class MoviesController < ApplicationController
   def index; end
 
+  def show
+    conn = Faraday.new(url: 'https://api.themoviedb.org') do |faraday|
+      faraday.headers['X-API-Key'] = ENV['MOVIEDB_API_KEY']
+    end
+    
+    movie = conn.get("/3/movie/#{params[:id]}?api_key=#{ENV['MOVIEDB_API_KEY']}&language=en-US")
+    @movie_details = JSON.parse(movie.body, symbolize_names: true)
+
+    @runtime = "#{@movie_details[:runtime] / 60}hrs, #{((@movie_details[:runtime].to_f / 60 - @movie_details[:runtime] / 60) * 60).to_i} minutes"
```

Comment:
Here's our super long one now.  What I've made is convoluted, and could be done better.